### PR TITLE
cnv 2.5: VMware imported VM name must not contain forward slash

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1884012

CP for 4.5, 4.6

### CODE REVIEW

**Review by atiratree — APPROVED** [review]

File: modules/virt-importing-vm-wizard.adoc [hunk 1]
```diff
@@ -63,6 +63,11 @@ ifdef::virt-importing-rhv-vm[]
 endif::[]
 ifdef::virt-importing-vmware-vm[]
 . Select a virtual machine or a template to import.
++
+[IMPORTANT]
+====
+The VMware VM name must not contain a forward slash (`/`). link:https://bugzilla.redhat.com/show_bug.cgi?id=1884012[(*BZ#1884012*)]
```
Comment (bergerhoffer) [inline modules/virt-importing-vm-wizard.adoc:69]: Is it really necessary to reference the BZ in the docs? Typically I don't think we do, outside of release notes.

Comment (apinnick) [reply modules/virt-importing-vm-wizard.adoc]: I removed it.

**Pull request merged**